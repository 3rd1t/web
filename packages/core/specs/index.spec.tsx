import React from "react"
import Index from "../pages/index"
import { render } from "@testing-library/react"

describe(" Index", () => {
  it("renders correctly", () => {
    expect(render(<Index />)).toMatchSnapshot()
  })
})
