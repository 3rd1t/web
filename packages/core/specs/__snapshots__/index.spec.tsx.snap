// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Index renders correctly 1`] = `
Array [
  <section
    className="bg-gray-100 "
    id="index"
  >
    <div
      className="relative h-screen"
    >
      <div
        className="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
      >
        <div
          className="flex flex-col items-start px-4 space-y-8 sm:pt-24 md:pt-28 lg:pt-32 xl:pt-40 xl:flex-row xl:space-y-0 xl:space-x-8"
        >
          <div
            className="flex flex-col items-center w-full"
          >
            <div
              className="space-y-4 text-left sm:text-center md:space-y-8 xl:space-y-12 lg:text-left"
            >
              <h2
                className="text-4xl font-extrabold leading-10 tracking-tight text-carbon-900 xl:text-5xl sm:leading-none md:text-5xl"
              >
                Portfolio analytics insights for everyone
              </h2>
              <div
                className="text-carbon-600 sm:text-lg sm:mx-auto md:text-xl lg:mx-0"
              >
                <p
                  className="flex flex-col xl:flex-row"
                >
                  Keeping track of all your assets and their performance is hard. Perfolio brings all information to one place and gives you access to the latest analytics methods in science.
                </p>
              </div>
              <div
                className="space-y-4 md:space-x-4 md:space-y-0"
              >
                <div
                  className="sm:flex sm:justify-center lg:justify-start"
                >
                  <div
                    className="rounded-md shadow"
                  >
                    <a
                      className="rounded-sm md:py-4 md:text-lg md:px-10 flex items-center justify-center w-full px-8 py-3 text-base font-medium leading-6 transition duration-200 ease-in-out focus:outline-none text-white bg-gray-900 hover:bg-carbon-800"
                      href="#"
                    >
                      Get started
                    </a>
                  </div>
                  <div
                    className="mt-3 sm:mt-0 sm:ml-3"
                  >
                    <a
                      className="rounded-sm md:py-4 md:text-lg md:px-10 flex items-center justify-center w-full px-8 py-3 text-base font-medium leading-6 transition duration-200 ease-in-out focus:outline-none text-carbon-900 hover:text-carbon-700"
                      href="Log in"
                    >
                      Log in
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <img
            alt="placeholder"
            src="https://via.placeholder.com/1280x720"
          />
        </div>
        <div
          className="absolute inset-x-0 bottom-0 flex justify-center mb-40"
          style={
            Object {
              "opacity": 0,
              "transform": "translateY(100px) translateZ(0)",
            }
          }
        >
          <div
            className="inline-flex w-10 h-10 p-2 text-gray-100 duration-500 transform rotate-45 shadow-xl lg:p-4 md:h-12 md:w-12 lg:h-16 lg:w-16 group bg-carbon-900 hover:scale-105 hover:-translate-y-4"
          >
            <div
              className="flex items-center justify-center transform -rotate-45"
            >
              <a
                href="#features"
              >
                <svg
                  fill="currentColor"
                  viewBox="0 0 20 20"
                >
                  <path
                    clipRule="evenodd"
                    d="M16.707 10.293a1 1 0 010 1.414l-6 6a1 1 0 01-1.414 0l-6-6a1 1 0 111.414-1.414L9 14.586V3a1 1 0 012 0v11.586l4.293-4.293a1 1 0 011.414 0z"
                    fillRule="evenodd"
                  />
                </svg>
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="bg-white"
    id="features"
  >
    <div
      className="flex flex-col justify-center"
    >
      <div
        className="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
      >
        <div
          className="md:text-center"
        >
          <h3
            className="text-3xl font-bold leading-8 tracking-tight text-carbon-900 sm:text-4xl sm:leading-10"
          >
            Why Perfolio?
          </h3>
          <p
            className="max-w-2xl mt-4 text-xl leading-7 text-carbon-500 md:mx-auto"
          >
            Keeping track of all your assets and their performance is hard. Perfolio brings all information to one place and gives you access to the latest analytics methods in science.
          </p>
        </div>
        <ul
          className="flex flex-col flex-wrap mx-auto my-24 md:flex-row"
        >
          <li>
            <div
              className="flex items-start p-3 space-x-1 md:space-x-4 md:w-1/2 xl:w-1/3"
            >
              <div
                className="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
              >
                <span
                  className="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
                >
                  <svg
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 21v-4m0 0V5a2 2 0 012-2h6.5l1 1H21l-3 6 3 6h-8.5l-1-1H5a2 2 0 00-2 2zm9-13.5V9"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                    />
                  </svg>
                </span>
              </div>
              <div
                className="flex-grow pl-4 lg:pl-0 lg:px-6"
              >
                <h2
                  className="text-lg font-medium leading-6 text-carbon-900"
                >
                  Independent
                </h2>
                <p
                  className="text-base leading-6 text-carbon-600"
                >
                  We are not part of any bank or insurance company. We give you an unbiased view of your portfolio, not selling any investment products.
                </p>
              </div>
            </div>
          </li>
          <li>
            <div
              className="flex items-start p-3 space-x-1 md:space-x-4 md:w-1/2 xl:w-1/3"
            >
              <div
                className="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
              >
                <span
                  className="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
                >
                  <svg
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16 8v8m-4-5v5m-4-2v2m-2 4h12a2 2 0 002-2V6a2 2 0 00-2-2H6a2 2 0 00-2 2v12a2 2 0 002 2z"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                    />
                  </svg>
                </span>
              </div>
              <div
                className="flex-grow pl-4 lg:pl-0 lg:px-6"
              >
                <h2
                  className="text-lg font-medium leading-6 text-carbon-900"
                >
                  Complex made easy
                </h2>
                <p
                  className="text-base leading-6 text-carbon-600"
                >
                  Everybody should have access to the latest analytics methods in science. Making these methods as simple and understandable as possible is part of our core business.
                </p>
              </div>
            </div>
          </li>
          <li>
            <div
              className="flex items-start p-3 space-x-1 md:space-x-4 md:w-1/2 xl:w-1/3"
            >
              <div
                className="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
              >
                <span
                  className="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
                >
                  <svg
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 12l2-2m0 0l7-7 7 7M5 10v10a1 1 0 001 1h3m10-11l2 2m-2-2v10a1 1 0 01-1 1h-3m-6 0a1 1 0 001-1v-4a1 1 0 011-1h2a1 1 0 011 1v4a1 1 0 001 1m-6 0h6"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                    />
                  </svg>
                </span>
              </div>
              <div
                className="flex-grow pl-4 lg:pl-0 lg:px-6"
              >
                <h2
                  className="text-lg font-medium leading-6 text-carbon-900"
                >
                  All in one place
                </h2>
                <p
                  className="text-base leading-6 text-carbon-600"
                >
                  Tired of visiting several websites to get an overview of your assets? Perfolio is the new home for your data.
                </p>
              </div>
            </div>
          </li>
          <li>
            <div
              className="flex items-start p-3 space-x-1 md:space-x-4 md:w-1/2 xl:w-1/3"
            >
              <div
                className="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
              >
                <span
                  className="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
                >
                  <svg
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M4 7v10c0 2.21 3.582 4 8 4s8-1.79 8-4V7M4 7c0 2.21 3.582 4 8 4s8-1.79 8-4M4 7c0-2.21 3.582-4 8-4s8 1.79 8 4m0 5c0 2.21-3.582 4-8 4s-8-1.79-8-4"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                    />
                  </svg>
                </span>
              </div>
              <div
                className="flex-grow pl-4 lg:pl-0 lg:px-6"
              >
                <h2
                  className="text-lg font-medium leading-6 text-carbon-900"
                >
                  Highest data quality
                </h2>
                <p
                  className="text-base leading-6 text-carbon-600"
                >
                  We cleanse and analyze data according to the highest standards. With our Premium Plan you get access to even better data quality.
                </p>
              </div>
            </div>
          </li>
          <li>
            <div
              className="flex items-start p-3 space-x-1 md:space-x-4 md:w-1/2 xl:w-1/3"
            >
              <div
                className="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
              >
                <span
                  className="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
                >
                  <svg
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M4 7v10c0 2.21 3.582 4 8 4s8-1.79 8-4V7M4 7c0 2.21 3.582 4 8 4s8-1.79 8-4M4 7c0-2.21 3.582-4 8-4s8 1.79 8 4m0 5c0 2.21-3.582 4-8 4s-8-1.79-8-4"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                    />
                  </svg>
                </span>
              </div>
              <div
                className="flex-grow pl-4 lg:pl-0 lg:px-6"
              >
                <h2
                  className="text-lg font-medium leading-6 text-carbon-900"
                >
                  Privacy and Security
                </h2>
                <p
                  className="text-base leading-6 text-carbon-600"
                >
                  We do not share your data with anyone else and store them only on servers within the EU. Privacy and security are our highest priority.
                </p>
              </div>
            </div>
          </li>
          <li>
            <div
              className="flex items-start p-3 space-x-1 md:space-x-4 md:w-1/2 xl:w-1/3"
            >
              <div
                className="z-10 inline-flex items-start justify-center text-gray-100 border-2 rounded bg-carbon-900 border-carbon-900"
              >
                <span
                  className="w-10 h-10 p-2 lg:p-3 xl:p-4 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:h-16 lg:w-16"
                >
                  <svg
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M3 10h18M7 15h1m4 0h1m-7 4h12a3 3 0 003-3V8a3 3 0 00-3-3H6a3 3 0 00-3 3v8a3 3 0 003 3z"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                    />
                  </svg>
                </span>
              </div>
              <div
                className="flex-grow pl-4 lg:pl-0 lg:px-6"
              >
                <h2
                  className="text-lg font-medium leading-6 text-carbon-900"
                >
                  Fair pricing
                </h2>
                <p
                  className="text-base leading-6 text-carbon-600"
                >
                  Start for free and if you want more, get the plan that suits you best. No hidden fees.
                </p>
              </div>
            </div>
          </li>
        </ul>
      </div>
    </div>
  </section>,
  <section
    id="team"
  >
    <div
      className="flex flex-col justify-center"
    >
      <div
        className="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
      >
        <h2
          className="text-3xl font-extrabold leading-8 tracking-tight text-center text-carbon-900 sm:text-4xl sm:leading-10"
        >
          Our Team
        </h2>
        <div
          className="flex flex-col items-center mt-10 md:flex-row"
        >
          <div
            className="text-center md:w-1/3 md:pr-8 md:py-8"
          >
            <div
              className="inline-flex items-center justify-center w-20 h-20 rounded-full md:h-32 md:w-32 bg-carbon-900"
            >
              <svg
                className="p-3 text-gray-100 stroke-current md:p-4"
                fill="none"
                viewBox="0 0 194 148"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 106.208L97.3883 17.8199L134.158 54.5894M185.07 41.8615L96.6814 130.25L59.9118 93.4803"
                  strokeWidth="25"
                />
              </svg>
            </div>
            <div
              className="flex flex-col items-center justify-center mt-4 space-y-2 text-center"
            >
              <h2
                className="text-lg font-medium leading-6 text-carbon-900"
              >
                Perfolio
              </h2>
              <p
                className="text-base leading-6 text-left md:text text-carbon-600 md:text-center"
              >
                At Perfolio we believe that investment decisions should always be based on as much information as possible. It is our mission to provide our users with the most relevant and accurate financial information available.
              </p>
            </div>
          </div>
          <div
            className="pt-4 mt-4 border-t border-carbon-300 md:w-2/3 md:pl-8 md:py-8 md:border-l md:border-t-0 md:mt-0"
          >
            <div
              className="flex flex-col items-center justify-center sm:flex-row sm:flex-wrap"
            >
              <div
                className="w-2/3 p-4 sm:w-1/2 lg:w-1/4"
              >
                <div
                  className="flex flex-col items-center justify-center text-sm "
                >
                  <img
                    alt="Nicolas Webersinke"
                    className="rounded-full"
                    src="/img/nico.jpeg"
                  />
                  <span
                    className="mt-4 font-semibold text-gray-900"
                  >
                    Nicolas Webersinke
                  </span>
                  <span
                    className="text-gray-600"
                  >
                    Product
                  </span>
                </div>
              </div>
              <div
                className="w-2/3 p-4 sm:w-1/2 lg:w-1/4"
              >
                <div
                  className="flex flex-col items-center justify-center text-sm "
                >
                  <img
                    alt="Andreas Thomas"
                    className="rounded-full"
                    src="/img/andreas.jpeg"
                  />
                  <span
                    className="mt-4 font-semibold text-gray-900"
                  >
                    Andreas Thomas
                  </span>
                  <span
                    className="text-gray-600"
                  >
                    Tech
                  </span>
                </div>
              </div>
              <div
                className="w-2/3 p-4 sm:w-1/2 lg:w-1/4"
              >
                <div
                  className="flex flex-col items-center justify-center text-sm "
                >
                  <img
                    alt="Mads Jordt"
                    className="rounded-full"
                    src="/img/mads.jpeg"
                  />
                  <span
                    className="mt-4 font-semibold text-gray-900"
                  >
                    Mads Jordt
                  </span>
                  <span
                    className="text-gray-600"
                  >
                    Sales
                  </span>
                </div>
              </div>
              <div
                className="w-2/3 p-4 sm:w-1/2 lg:w-1/4"
              >
                <div
                  className="flex flex-col items-center justify-center text-sm "
                >
                  <img
                    alt="Kevin Kohler"
                    className="rounded-full"
                    src="/img/kevin.jpeg"
                  />
                  <span
                    className="mt-4 font-semibold text-gray-900"
                  >
                    Kevin Kohler
                  </span>
                  <span
                    className="text-gray-600"
                  >
                    Marketing
                  </span>
                </div>
              </div>
            </div>
            <p
              className="mt-8 text-center text-carbon-600"
            >
              Our mutual interest in programming, scientific research, and financial investments are the backbone of Perfolio.
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="bg-white"
    id="subscribe"
  >
    <div
      className="flex flex-col justify-center"
    >
      <div
        className="container px-4 py-10 mx-auto md:py-12 lg:py-16 xl:py-20 xl:px-0"
      >
        <div
          className="container flex flex-col justify-around mx-auto lg:flex-row"
        >
          <div>
            <h2
              className="text-2xl font-semibold leading-8 text-carbon-900 font-display sm:text-3xl sm:leading-9"
            >
              Curious for more?
            </h2>
            <p
              className="max-w-2xl mt-2 text-base leading-6"
            >
              <span
                className="text-carbon-600"
              >
                Follow us on our roadmap and on 
              </span>
              <a
                href="https://github.com/perfolio"
                target="blank"
              >
                github
              </a>
              .
            </p>
          </div>
          <form
            className="mt-6"
          >
            <div
              className="sm:flex"
            >
              <input
                className="block w-full px-4 py-3 text-base leading-6 bg-white border rounded shadow-none placeholder-carbon-600 border-carbon-300 sm:max-w-xs focus:outline-none focus:border-carbon-700"
                placeholder="Enter your email"
                type="email"
              />
              <button
                className="relative block w-full px-6 py-3 mt-4 text-base font-semibold leading-6 text-white transition duration-150 ease-in-out border border-transparent rounded shadow-md bg-carbon-900 sm:mt-0 sm:h-auto sm:ml-4 sm:w-auto hover:bg-carbon-800 focus:outline-none focus:bg-carbon-800 hover:bg-carbon-600"
              >
                Subscribe
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </section>,
]
`;
